app/views/publishers/home.html.slim

```diff
@@ -28,64 +28,28 @@ script type="text/html" id="confirm_default_currency_modal_wrapper"
-          .float-left
-            = image_tag("bat-logo@2x.png", class: "", width: 60, height: 57)
-          .float-left.amounts
-            .bat
-              span.bat-amount#bat_amount= publisher_humanize_balance(current_publisher, "BAT")
-              span.bat-code= " BAT"
-            .converted#converted_amount
-              = publisher_converted_balance(current_publisher)
-            .next_deposit_date
-              span = t ".next_deposit_date"
-              span= next_deposit_date
-      - if promo_running?
-        .col
-          .promo-panel
-            - if current_publisher.promo_status(promo_running?) == :inactive
-              .promo-panel-item.promo-panel-item-info
-                #promo-limited-time= t("promo.activate.limited_time_promo")
-                h3#promo-title= t("promo.activate.title")
-                #promo-referring-fans= t("promo.activate.for_referring_your_fans")
-              .promo-panel-item.promo-panel-item-button
-                = form_for current_publisher, { method: :get, url: promo_registrations_path } do |f|
-                  = f.submit(t("promo.activate.button-dashboard").upcase, class: "btn btn-primary", id: "activate-promo-dashboard-button" )
-            - elsif current_publisher.has_verified_channel?
-              .promo-flex-col
-                .promo-flex-row.promo-flex-row-main
-                  .promo-flex-col.promo-panel-title
-                    .promo-panel-title-item
-                      =t("promo.dashboard.referral").upcase
-                    .promo-panel-title-item
-                      =t("promo.dashboard.promo").upcase
-                  .promo-flex-col.promo-flex-col-downloaded
-                    .promo-panel-header
-                      =t("promo.dashboard.downloaded").upcase
-                    .promo-panel-number
-                      = total_referral_downloads(current_publisher)
-                  .promo-flex-col.promo-flex-col-confirmed
-                    .promo-panel-header
-                      =t("promo.dashboard.confirmed").upcase
-                    .promo-panel-number
-                      = confirmed_referral_downloads(current_publisher)
-                .promo-flex-row.promo-flex-row-sub
-                  =t("promo.dashboard.check_statement")
-            - else
-              .promo-panel-item.promo-panel-item-alert
-                #promo-panel-alert
-                  = t("promo.dashboard.add_channel")
-      - elsif current_publisher.channels.empty?
-        .col
-          .add-channel-cta
-            = t ".add_channel_cta"
-    .row
+        .dashboard-panel--wrapper
+          h4.dashboard-panel--header#dashboard_brave_header
+            .row
+              .col.col-md-8
+                = t ".brave_fan_contributions"
+              .col.col-md-4#dashboard_info_box
+                = image_tag("small-calendar@1.5x.png")
+                = " #{Date.today.beginning_of_month.strftime('%m/%d')} - #{Date.today.strftime('%m/%d')}"
```

Comment:
I would argue that this should most likely be in a helper, and that the strftime should be internationalized because different countries display their dates in different ways.